ilastik/applets/pixelClassification/pixelClassificationGui.py

```diff
@@ -718,46 +702,42 @@ def setPredLayerName(n, predictLayer_=predictLayer, initializing=False):
                 ref_label.pmapColorChanged.connect(setLayerColor)
                 ref_label.nameChanged.connect(setPredLayerName)
                 layers.append(predictLayer)
-        
-        self.handleLabelSelectionChange()
         return layers
 
-    def toggleInteractive(self, checked):
-        logger.debug("toggling interactive mode to '%r'" % checked)
+    def hasFeatures(self) -> bool:
+        feature_images_slot = self.topLevelOperatorView.FeatureImages
+        return feature_images_slot.ready() and feature_images_slot.meta.shape != None
```

Comment:
```suggestion
        return feature_images_slot.ready() and feature_images_slot.meta.shape is not None
```

Also, this method can be a property.